src/core/icons/fileIcons.ts

```diff
@@ -880,7 +880,30 @@ export const fileIcons: FileIcons = {
     {
       name: 'graphql',
       fileExtensions: ['graphql', 'gql'],
-      fileNames: ['.graphqlconfig'],
+      fileNames: [
+        '.graphqlconfig',
+        'graphql.config.ts',
+        'graphql.config.cts',
+        'graphql.config.mts',
+        'graphql.config.js',
+        'graphql.config.cjs',
+        'graphql.config.mjs',
+        'graphql.config.json',
+        'graphql.config.yaml',
+        'graphql.config.yml',
+        'graphql.config.toml',
+        '.graphqlrc',
+        '.graphqlrc.ts',
+        '.graphqlrc.cts',
+        '.graphqlrc.mts',
+        '.graphqlrc.js',
+        '.graphqlrc.cjs',
+        '.graphqlrc.mjs',
+        '.graphqlrc.json',
+        '.graphqlrc.yml',
+        '.graphqlrc.yaml',
+        '.graphqlrc.toml',
+      ],
       patterns: {
         graphql: FileNamePattern.Ecmascript,
```

Comment:
@3w36zj6 would you mind changing the pattern to `Cosmiconfig`? Then we don't need to adjust so many file names as they are already covered by that pattern.

See https://github.com/material-extensions/vscode-material-icon-theme/blob/92fdcb16cac0f5f42c43037b7dcc2cdcdb6869ab/src/core/patterns/patterns.ts#L54